ethportal-peertest/src/scenarios/find.rs

```diff
@@ -81,5 +109,12 @@ pub fn test_trace_recursive_find_content_local_db(
 
     let result = make_ipc_request(&peertest.bootnode.web3_ipc_path, &request).unwrap();
     assert_eq!(result["content"], json!(HISTORY_CONTENT_VALUE.to_string()));
-    assert_eq!(result["route"], json!([]));
+    assert_eq!(
+        result["trace"]["received_content_from_node"],
```

Comment:
This is an example of what I don't like about the `Value` type. Having a more specific type is easier to work with.